terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -878,6 +882,16 @@ def qq(*words):
                                       from_port=3,  # Destination Unreachable
                                       to_port=4)  # Fragmentation required DF-flag set
                 ]
+            },
+            'elasticsearch': {
```

Comment:
Since there is no reference to this in the `gitlab` component this can be moved to the main component. Generally speaking, you should minimize the number of resources added to the main component.